examples/connector/java/src/main/java/connector/ConnectorServiceImpl.java

```diff
@@ -81,101 +87,73 @@ public void schema(SchemaRequest request, StreamObserver<SchemaResponse> respons
     @Override
     public void update(UpdateRequest request, StreamObserver<UpdateResponse> responseObserver) {
         Map<String, String> configuration = request.getConfigurationMap();
-        String state_json = request.hasStateJson() ? request.getStateJson() : "{}";
+        String stateJson = request.hasStateJson() ? request.getStateJson() : "{}";
         Selection selection = request.hasSelection() ? request.getSelection() : null;
 
         ObjectMapper mapper = new ObjectMapper();
         UpdateResponse.Builder responseBuilder = UpdateResponse.newBuilder();
 
         try {
-            State state = mapper.readValue(state_json, State.class);
+            State state = mapper.readValue(stateJson, State.class);
 
             // -- Send a log message
-            responseBuilder.clear();
-            responseObserver.onNext(responseBuilder
-                    .setLogEntry(LogEntry.newBuilder()
-                            .setLevel(LogLevel.INFO)
-                            .setMessage("Sync STARTING")
-                            .build())
-                    .build());
+            System.out.println(new LogEntry(INFO, "[Update]: Sync STARTING", SDK_CONNECTOR));
```

Comment:
Could we use a proper logger instead of `System.out.println()`? And then perhaps we could also eliminate the new `LogEntry` class too?

Reply:
updated